pytest_bdd/steps.py

```diff
@@ -183,7 +134,9 @@ def lazy_step_func():
         if converters:
             step_func.converters = lazy_step_func.converters = converters
 
-        lazy_step_func = pytest.fixture(scope=scope)(lazy_step_func)
+        step_func.target_fixture = lazy_step_func.target_fixture = target_fixture
+
+        lazy_step_func = pytest.fixture()(lazy_step_func)
```

Comment:
Isn't this line still making the step function a fixture? I thought this was one of the main thing to remove.

Reply:
is it not the step implementation is a fixure and this is how you get them by importing all in the conftest?